epyqlib/devicetreeview.py

```diff
@@ -231,26 +231,30 @@ def add_device(self, bus, device=None):
         return device
 
     def change_device_number(self, node):
-        pcan_bus = can.interface.Bus(bustype=node.interface, channel=node.channel)
-        current_device_number = pcan_bus.get_device_number()
-        new_device_number, ok = QInputDialog.getInt(
-            None,
-            *(("Device ID",) * 2),
-            current_device_number,
-            0,
-            255,
-        )
+        pcan_bus = None
+        try:
+            pcan_bus = can.interface.Bus(bustype=node.interface, channel=node.channel)
```

Comment:
I think the `None` bit, both above and below, can be skipped and this assignment can just move outside.

Reply:
Right again. I got distracted by an error I was able to reproduce on line 236 that involved quick mouse clicking. It isn't necessary to have line 236 within the try since that error gets handled properly anyways. This is the kind of stuff I would have to do in my old app since the error wouldn't be handled properly. Thanks.